Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -228,16 +238,19 @@ private object DeserializeWithIonConstructor(ConstructorInfo ionConstructor, IIo
 
             reader.StepIn();
 
-            // Deserialize Ion and organize deserialized results into three categories:
+            // Deserialize Ion and organize deserialized results into four categories:
             // 1. Values to be passed into the Ion constructor.
-            // 2. Properties to be set after construction.
-            // 3. Fields to be set after construction.
+            // 2. Values to be set via annotated setters after construction.
```

Comment:
Can you specifically mention set via methods?

Reply:
Done.